app/models/nic/base.rb

```diff
@@ -22,7 +22,7 @@ class Base < ActiveRecord::Base
     validates :mac, :uniqueness => {:scope => :virtual},
               :if => Proc.new { |nic| nic.managed? && nic.host && nic.host.managed? && !nic.host.compute? && !nic.virtual? }, :allow_blank => true
     validates :mac, :presence => true,
-              :if => Proc.new { |nic| nic.managed? && nic.host && nic.host.managed? && !nic.host.compute? &&!nic.virtual? }
+              :if => Proc.new { |nic| nic.managed? && nic.host && nic.host.managed? && !nic.host.compute? && !nic.virtual? && SETTINGS[:unattended] }
```

Comment:
Curious, so `nic.host.managed?` is actually true, and I see it in the request parameters I pasted earlier.  I wonder if that's correct and when in !unattended mode it would make more sense to say all hosts are unmanaged?  Should there be any differences between an unmanaged host in unattended mode vs !unattended mode?

It's an open question, perhaps @ares has thoughts?